organization/_posts/2022/2022-07-07-generators.md

```diff
@@ -0,0 +1,26 @@
+yout: plain_toc
```

Comment:
I believe you meant "layout"

Reply:
Right but even more: 
---
title: "HSF Generator WG Meeting #23, 7 July 2022"
layout: plain_toc
---
I am updating it now.